zimbra-x-webclient.adoc

```diff
@@ -0,0 +1,106 @@
+= Installing {product-short} X Webclient
+
+Optionally you can install {product-short} X Webclient, which is currently in beta stage and you can provide early feedback to {product-short}.
+Below are the steps to setup {product-short} X Webclient on {product-short}
+
+. Run {product-short} X Webclient on any of the mailbox servers
+
+	* Make sure latest version of **NodeJS** and **NPM** is installed
+	* In case of multinode environment make sure you are selecting a node which is currently working as a mailbox node
+	* Clone respository of {product-short} X Webclient by executing `git clone https://github.com/Zimbra/zm-x-web.git`
+	* If you want to do any customizations on top of {product-short} X Webclient then you can do as shown here https://github.com/Zimbra/zm-x-web/wiki/Customising-Zimbra-X-Webclient[Customising-Zimbra-X-Webclient]
+	* Install npm modules by executing `npm install`
+	* Create a production build by executing `npm run build`.  If customisations were made in a client directory other that `default`, e.g. in the `client/foo` directory, then specify the client directory name as an environment variable on the command line like `CLIENT=foo npm run build`
+	* `PORT=9090 npm run serve` (We are specifying port 9090 to make sure it doesn't conflict with other used ports in {product-short})
+
+. Configure nginx to route requests properly
+	* add below line in `conf/nginx/templates/nginx.conf.web.template` file at the end of file where other files are getting included
```

Comment:
Mention root path /opt/zimbra/  in location `conf/nginx/templates/nginx.conf.web.template`

Reply:
Fixed